### DISCUSSION TITLE
DK Starting area no rune forge spell

### DISCUSSION THREAD

**Discussion by crprowse** [azerothcore/azerothcore-wotlk#19942] [General]
Forgive me if I am posting this wrong or in the wrong area.
First Thank You for everything you are doing, I am enjoying your project beyond measure. I do however have an issue with starting a DK and completing the second quest and not receiving the runeforge spell. I can set it with .learn but was wondering if there is a fix I can apply? Again Thank you for everything

**Comment by sogladev** [comment]
Are you using liyunfan1223/azerothcore-wotlk? This is reported to not be an issue on stock AC. 

See comment on mod-playerbots linked below issue

- https://github.com/liyunfan1223/mod-playerbots/issues/531#issuecomment-2350313035

**Comment by crprowse** [comment]
Thank you Thank you!!!

I made the corrections to the Playerquest.cpp and it works Awesome!

Thank you for everything!


Thanks,

Chris



[cid:4ca6e1b6-790d-4576-b5bb-ca32ea0af8f2]

Christopher Prowse

Server Manager

Little Rock School District

Information Services

***@***.******@***.***>

www.lrsd.org<http://www.lrsd.org/>

501-447-1289



________________________________
